examples/watch.js

```diff
@@ -0,0 +1,57 @@
+//
+// Watch changes to a Deployments in a specific Namespace.
+//
+const Client = require('../lib').Client;
+const config = require('../lib').config;
+const JSONStream = require('json-stream');
+
+const deploymentManifest = require('./nginx-deployment.json');
+
+async function cleanup(client) {
+  try {
+    await client.apis.apps.v1.namespaces('default').deployments(deploymentManifest.metadata.name).delete();
+  } catch (err) { }
+}
+
+async function triggerEvents(client) {
+  //
+  // Trigger some events
+  //
+  await cleanup(client);
+  for (let count = 0; count < 3; count++) {
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Why do you need to wait here?

Reply:
I want to delay 1 second before executing the next line ("`sleep 1`")